From: Georges Khaznadar <georgesk@debian.org>
Date: Sat, 3 Sep 2016 17:57:10 +0200
Subject: Remove download target for noAvhttp

In Debian we don't load anything dynamically from the net while package
built.
---
 CMakeModules/download_avhttp.cmake | 27 ---------------------------
 1 file changed, 27 deletions(-)

diff --git a/CMakeModules/download_avhttp.cmake b/CMakeModules/download_avhttp.cmake
index abc52b7e..1827273d 100644
--- a/CMakeModules/download_avhttp.cmake
+++ b/CMakeModules/download_avhttp.cmake
@@ -35,31 +35,4 @@
 #-----</configure>-----------------------------------------------------------------------------------
 
 
-# Where the library is to be installed.
-set( PREFIX ${DOWNLOAD_DIR}/avhttp )
-
-if( KICAD_SKIP_BOOST )
-    set( AVHTTP_DEPEND "" )
-else()
-    set( AVHTTP_DEPEND "boost" )
-endif()
-
-
-# Install the AVHTTP header only library ${PREFIX}
-ExternalProject_Add( avhttp
-    PREFIX          ${PREFIX}
-    DOWNLOAD_DIR    ${DOWNLOAD_DIR}     # no true download yet
-
-    # grab it from a local zip file for now, cmake caller's source dir
-    URL             ${CMAKE_CURRENT_SOURCE_DIR}/avhttp-master.zip
-    DEPENDS         ${AVHTTP_DEPEND}
-
-    CONFIGURE_COMMAND ""
-
-    BUILD_COMMAND   ""
-    INSTALL_COMMAND ${CMAKE_COMMAND} -E copy_directory <SOURCE_DIR> <INSTALL_DIR>
-    )
-
-
-set( AVHTTP_INCLUDE_DIR  "${PREFIX}/include" CACHE FILEPATH "AVHTTP include directory" )
 mark_as_advanced( AVHTTP_INCLUDE_DIR )
